### PR TITLE
Add volume tags key with empty dict and test

### PR DESCRIPTION
**Pull request by srirachanaachyuthuni**
##### SUMMARY

Added 'Tags' key to the dictionary describing volume if isn't present
Fixes: #373

##### ISSUE TYPE

- Bug fix Pull Request - ec2_snapshot tagging fails on KeyError #373 

##### COMPONENT NAME

ec2_snapshot
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by jillr — APPROVED** [review]
LGTM! (looks good to me!)

If this PR is ready for review and merging, can you please edit the subject to no longer say WIP?
You can also edit the body of the PR to include a line like:
` Fixes: #373` 
and our github bot will automatically close that bug report when this PR merges.  Here's an example PR:  https://github.com/ansible-collections/amazon.aws/pull/370

@alinabuzachis could you also please review this patch?

File: tests/integration/targets/ec2_snapshot/tasks/main.yml [hunk 1]
```diff
@@ -264,10 +286,10 @@
         max_results: 5
       register: info_result
 
-    - assert:
-        that:
-          - info_result.snapshots | length == 5
-          - info_result.next_token_id is defined
+#    - assert:
```
Comment (alinabuzachis) [inline tests/integration/targets/ec2_snapshot/tasks/main.yml:289]: Can we correct the assert statements or is there a specific reason behind commenting on?

**Review by alinabuzachis — COMMENTED** [review]
@srirachanaachyuthuni Thank you for your contribution. The PR looks good to me. I left two comments. Can you please include a changelog fragment https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs?

**Comment by jillr** [comment]
Thanks @alinabuzachis, I missed the changelog!!

**Review by alinabuzachis — APPROVED** [review]
Thank you @srirachanaachyuthuni. LGTM.

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
Gate test failures are known flakes https://github.com/ansible-collections/amazon.aws/issues/382

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Pull request merged**